<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CostmapViewer</class>
 <widget class="QWidget" name="CostmapViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>costmapviewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbWarp">
       <property name="text">
        <string>Warp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CVSimpleViewer" name="disparityImage"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CVSimpleViewer" name="originalLeft"/>
       </item>
       <item>
        <widget class="CVSimpleViewer" name="originalRight"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CVSimpleViewer" name="disparityZoom"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="CVSimpleViewer" name="windowOriginal"/>
       </item>
       <item>
        <widget class="CVSimpleViewer" name="disparityRangeViewer"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="patch_info">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>costs</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="table"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>stat</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="stattable"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rowinfo">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MiniPlot" name="plot" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CVSimpleViewer</class>
   <extends>QWidget</extends>
   <header>cvsimpleviewer.h</header>
  </customwidget>
  <customwidget>
   <class>MiniPlot</class>
   <extends>QWidget</extends>
   <header>miniplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
